### ISSUE TITLE
Add images to Image file

### ISSUE THREAD

**Issue by Victorybrown** [Victorybrown/DezignHers#2]
![dezignhers2](https://user-images.githubusercontent.com/64163022/139566227-5d8a8789-d3a1-4f4a-9cda-f53a4ddef454.png)

![dezignhers](https://user-images.githubusercontent.com/64163022/139566233-fc0c0f33-89a1-42fa-9724-3ef005b72e04.png)

![dh1](https://user-images.githubusercontent.com/64163022/139566239-14318652-103d-451d-9e33-69f97beda7d2.png)

![dezignhers1](https://user-images.githubusercontent.com/64163022/139566244-74f8d353-8829-4dfb-a1a5-952850bb0c4f.png)

**Comment by PrachiAgarwal19** [comment]
Hi! I would like to work on this. Just to confirm , I need to add the above images in  the image folder, right?

**Comment by Victorybrown** [comment]
Yeah. I'm sorry i couldn't find your comment on the issue. But yes thats it.
